openshift/templates/email-verification-agent/email-verification-agent-deploy.yaml

```diff
@@ -30,11 +75,13 @@ objects:
     metadata:
       name: ${NAME}
       labels:
-        name: ${NAME}
-        app: ${APP_GROUP}
+        app: ${APP_NAME}${SUFFIX}
+        name: ${NAME}${SUFFIX}
         app-group: ${APP_GROUP}
+        role: ${ROLE}
+        env: ${TAG_NAME}
     spec:
-      host: ${APPLICATION_DOMAIN}
+      host: ${APPLICATION_AGENT_DOMAIN}
       port:
         targetPort: ${NAME}-agent
```

Comment:
Nit.: mind changing the suffix from `-agent` to `-public`? It will just read a bit weird when created as it will be `email-verification-agent-agent`... 😅

Reply:
Changing `APPLICATION_DOMAIN` may have unexpected consequences too.  `APPLICATION_DOMAIN` is a keyword in the scripts that has some "special" processing associated to it when the `.param` files are generated; such as automatically updating the URL for `dev`, `test`, `prod`, and uncommenting it in the associated `.param` files.

Reply:
ya it seems like it changes APPLICATION_DOMAIN to the old pathfinder domain, maybe we should chat and update the special processing

Reply:
changed -agent to -public

Reply:
reverted to APPLICATION_DOMAIN